graylog2-server/src/main/java/org/graylog/plugins/views/search/rest/MessagesResource.java

```diff
@@ -147,6 +157,36 @@ private MessagesRequest fillInIfNecessary(MessagesRequest requestFromClient) {
         return asyncRunner.apply(chunkConsumer -> exporter(searchId, searchTypeId).export(command, chunkConsumer));
     }
 
+    @ApiOperation("Retrieve results for export job")
+    @GET
+    @Path("job/{exportJobId}/{filename:.*}")
```

Comment:
Thanks for the heads up! Seems like some rebase went wrong when basing this on #10675, fixed it.

1. The `filename` path parameter is not used, it's just there because in my tests Jersey always expects the catch-all part of the path to have a parameter name, otherwise it will be ignored. The default browser behavior is to take this part of the URL as the filename for the download, so we do not need to set it explicitly.
2. The `filename` field is part of the `ResultFormat` class which is also used for the export through the already existing `POST` request. In there, it _was_ used (until this PR) to create the filename for the downloaded file (which we do not need for the export through the export job due to 1.), I would like to use this in a follow-up PR to set the `Content-Disposition` header for the existing `POST`-export.

Reply:
@dennisoelkers Thanks for the clarification! Sounds great! :+1: